homeworks/hw12/card_desk/cards_deck.py

```diff
@@ -0,0 +1,53 @@
+import random
+
+
+class Card:
+    def __init__(self):
+        self.cards = []
+        self.number_list = list(range(1, 14))
+        self.mast_list = ['Hearts', 'Diamonds', 'Clubs', 'Spades']
+
+        for mast in self.mast_list:
+            for number in self.number_list:
+                self.cards.append((mast, number))
+
+        self.cards.append(('Joker', '1'))
+        self.cards.append(('Joker', '2'))
+
+
+class CardsDeck:
+    def __init__(self):
+        self.card = Card()
+        self.deck = self.card.cards[:]
```

Comment:
Something like this:
```suggestion
        for mast in ['Hearts', 'Diamonds', 'Clubs', 'Spades']:
            for number in range(1, 14):
                self.cards.append(Card(mast, number))
         ... and jokers
```